hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/RateProperties.java

```diff
@@ -0,0 +1,98 @@
+package pl.allegro.tech.hermes.consumers.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import pl.allegro.tech.hermes.consumers.consumer.rate.calculator.RateCalculatorParameters;
+
+@ConfigurationProperties(prefix = "consumer.rate")
+public class RateProperties {
```

Comment:
what do you think of: `RateProperties` -> `ConsumerRateProperties`? Or maybe it will be redundant 🤔

Reply:
`*_RATE_*` properties are only present in consumers, so I would not add `Consumer` prefix to avoid redundancy